src/main/java/de/zalando/aruha/nakadi/service/StrategiesRegistry.java

```diff
@@ -9,20 +9,13 @@
 public class StrategiesRegistry {
 
     public static final PartitionStrategyDescriptor HASH_PARTITION_STRATEGY =
-            new PartitionStrategyDescriptor("hash", "This strategy will use the event " +
```

Comment:
oh, I apologize! i didn't know it was already being exposed with the current implementation... 
my suggestion was more on the pragmatic side. 

do you really think we should remove this after all?